tasks/76_deep-object-clone/deep-object-clone.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @description
+ * Copies an argument object
+ * @example
+ * // returns { a: 1, b: { c:10 } }
+ * deepClone({ a: 1, b: { c:10 } })
+ * @param {Object} obj - object to copy
+ * @return {Object} - copy itself
+ */
+export function deepClone(obj) {
+  const copy = {};
+  for (let key in obj) {
+    if (obj.hasOwnProperty(key)) {
+      if (Array.isArray(obj[key])) {
```

Comment:
Are you sure that you need Array's check?

Reply:
well, I really don't need it, yet it works kinda faster with slice (on my machine 9ms with slice and 17ms without). Should I remove array check?